server/lessons/en/scrape-multiple-pages.html

```diff
@@ -77,22 +77,16 @@ <h2>Extract data from multiple HTML pages</h2>
   <p>
     We still need to extract the information we want from all this source code. We can do so with <b>HTML to Table</b>. It will automatically combine the results from all pages into one table.
   <p class="instruction-detail">
-    Learn how to use <b>HTML to Table</b> and XPath in this tutorial:<br> <a href="https://app.workbenchdata.com/lessons/scrape-using-xpath" class="highlight">Scrape a webpage to extract or monitor content</a>
+    Learn how to use <b>HTML to Table</b> and XPath in this tutorial:<br><a href="scrape-using-xpath" class="highlight">Scrape a webpage to extract or monitor content</a>
```

Comment:
This change hasn't been ported to Greek (the Greek file does not have `https://app.workbenchdata.com` but has `/lessons/el/`)